packages/web-components/fast-components/src/divider/divider.styles.ts

```diff
@@ -25,4 +25,12 @@ export const dividerStyles: (
             border: none;
             border-top: calc(${strokeWidth} * 1px) solid ${neutralStrokeDividerRest};
         }
+
+        :host([orientation="vertical"]) {
+            height: 100%;
+            margin: 0 calc(${designUnit} * 1px);
+            border: none;
```

Comment:
`border: none` is inherited from the cascade and isn't needed here.

Reply:
I guess it can we removed from the existing style also then